package.json

```diff
@@ -73,14 +73,14 @@
   },
   "peerDependencies": {
     "immutable": "4.0.0-rc.10",
-    "lattice": "^0.93.5",
-    "lodash": "^4.17.21",
-    "luxon": "^1.27.0",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
-    "react-redux": "^7.2.0",
-    "redux": "^4.0.0",
-    "redux-reqseq": "^0.6.1"
+    "lattice": ">=0.94.0",
+    "lodash": ">=4.17.21",
+    "luxon": ">=2.0.0",
```

Comment:
I think there was a reason why we didn't upgrade to luxon 2. iirc the material ui pickers still uses 1.x

https://material-ui-pickers.dev/getting-started/installation

Reply:
I see ... will revert (and might need to revert other projects as well).